dashboard/test/helpers/workshop_survey_results_helper_test.rb

```diff
@@ -107,6 +107,23 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
     assert_equal 3.67, result_hash[:how_clearly_presented]
   end
 
+  test 'averaging across multiple surveys' do
+    workshop_1 = create(:pd_workshop, :local_summer_workshop, num_sessions: 1, num_facilitators: 2, num_completed_surveys: 5)
+    workshop_2 = create(:pd_workshop, :local_summer_workshop, num_sessions: 1, num_facilitators: 3, num_completed_surveys: 10)
+
+    workshop_2.survey_responses.each do |response|
+      response.update_form_data_hash({how_clearly_presented: {workshop_2.facilitators.first.name => 'Not at all clearly', workshop_2.facilitators.second.name => 'Not at all clearly', workshop_2.facilitators.third.name => 'Not at all clearly'}})
+      response.save
+    end
+
+    # With 10 people saying "Not at all clearly" to 3 facilitators, and 5 people saying
+    # "Extremely Clearly" to 2 facilitators, we'd expect the answer to be
+    # [(10 * 3 * 1) + (5 * 2 * 5)] / (10 + 30)
```

Comment:
Let's say I have two workshops. One workshop has a small number of positive reviews for all facilitators. Another has a large number of negative reviews for most facilitators. Deriving the average score is equal to (sum of all facilitator scores) / (sum of the total number of faciltator responses)

The comment was more for me to keep my head straight when writing the test. If it makes things more confusing, I'll remove it